repository/model/settings.go

```diff
@@ -75,17 +79,20 @@ func ToSvcSettings(sv []SettingsValue) (svcmodel.Settings, error) {
 				return svcmodel.Settings{}, err
 			}
 		case keySlack:
-			if err := json.Unmarshal(settingsValue.Value, &s.Slack); err != nil {
+			if err := json.Unmarshal(settingsValue.Value, &slackSettings); err != nil {
 				return svcmodel.Settings{}, err
 			}
 		case keyGithub:
-			if err := json.Unmarshal(settingsValue.Value, &s.Github); err != nil {
+			if err := json.Unmarshal(settingsValue.Value, &githubSettings); err != nil {
```

Comment:
I need to first unmarshal it into a struct from the repository model because this model contains the JSON tags.